crates/forge/tests/e2e/trace_data.rs

```diff
@@ -0,0 +1,39 @@
+use forge_runner::trace_data::TRACE_DIR;
+
+use crate::e2e::common::runner::{setup_package, test_runner};
+
+#[test]
+fn simple_package_save_trace() {
+    let temp = setup_package("simple_package");
+    let snapbox = test_runner();
+    snapbox
+        .current_dir(&temp)
+        .arg("--save-trace-data")
+        .assert()
+        .code(1);
+
+    assert!(temp
+        .join(TRACE_DIR)
+        .join("simple_package::tests::test_fib.json")
+        .exists());
+    assert!(!temp
+        .join(TRACE_DIR)
+        .join("tests::test_simple::test_failing.json")
+        .exists());
```

Comment:
you aren't actually testing any contents, are you?

Reply:
I feel like semantics of the trace content are integration tested by @piotmag769. I don't feel like explicitly asserting the content here would make a very strong test above what we have. If you have an idea how we can test it i am open to add something here.

Reply:
I guess we can test content of one of the files - it doesn't test the trace collection mechanism, but we will be sure someone doesn't mess up sth in the saving to file section of the code in the future

Reply:
I've added the file parsing to the test.